jupyter-notebooks/orders_basemaps/requests_order_basemaps.ipynb

```diff
@@ -518,57 +772,206 @@
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "success\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "success\n",
+      "success\n",
+      "success\n",
+      "success\n",
+      "success\n",
+      "success\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "running\n",
+      "success\n",
+      "success\n",
+      "success\n"
+     ]
+    }
+   ],
    "source": [
-    "param_res = session.post(\n",
-    "    ORDERS_API_URL, json=order_params, auth=auth, headers=headers)\n",
+    "order_urls = batch_list_and_place_orders(\n",
+    "    quad_ids, polygon_search_order_params, auth)\n",
     "\n",
-    "if param_res.status_code < 400:\n",
-    "    basemaps = param_res.json()\n",
-    "else:\n",
-    "    param_res.raise_for_status()"
+    "for order in order_urls:\n",
+    "    poll_for_success(order)"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 15,
+   "execution_count": 401,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "basemaps-to-cloud/25486588-0f1b-4627-aa66-b27a9c765be9/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_provenance_vector_merge_clip.zip\n",
-      "basemaps-to-cloud/25486588-0f1b-4627-aa66-b27a9c765be9/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_metadata_merge_clip.json\n",
-      "basemaps-to-cloud/25486588-0f1b-4627-aa66-b27a9c765be9/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_ortho_udm_merge_clip.tif\n",
-      "basemaps-to-cloud/25486588-0f1b-4627-aa66-b27a9c765be9/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_merge_clip.tif\n",
-      "basemaps-to-cloud/25486588-0f1b-4627-aa66-b27a9c765be9/global_monthly_2022_01_mosaic/global_monthly_2022_01_mosaic_provenance_raster_merge_clip.tif\n",
-      "basemaps-to-cloud/25486588-0f1b-4627-aa66-b27a9c765be9/manifest.json\n"
+      "success\n"
      ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<Response [200]>"
+      ]
+     },
+     "execution_count": 401,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "r = requests.get(order_url, auth=session.auth)\n",
-    "response = r.json()\n",
-    "results = response['_links']['results']\n",
-    "print(\"\\n\".join([r['name'] for r in results]))"
+    "poll_for_success(order_url)"
```

Comment:
I think I'm confused what's supposed to be happening here. Why do we poll in this cell and the cell above? Do we need both?